### ISSUE TITLE
do you have any finished app created with this lib?

### ISSUE THREAD

**Issue by seniorivn** [benlau/quickandroid#29]
Hi, great work you've done, do you have any serious app created with it? I'd like to see it in the market/appstore

**Comment by benlau** [comment]
Currently I don't have any app using this library is available on Google Play yet. However, there has a demonstration apk available to download at 

https://drone.io/github.com/benlau/quickandroid/files

**Comment by seniorivn** [comment]
yeah, i already compiled it, looks good.
But you are creating this library for apps that you are actually developing?

**Comment by benlau** [comment]
yes, it is original created for an app. But the development is suspended and we moved the focus to another product.

**Comment by shirayukikitsune** [comment]
Sorry if reviving the topic, but I would like to list a WIP application made for a client of mine, which is already published on Google Play. It is portuguese only, but the attached screenshots should give you an idea of how it runs.

Its name is [APOIE](https://play.google.com/store/apps/details?id=br.com.eletricistaconsciente.apoie).

**Comment by seniorivn** [comment]
@shirayukikitsune thanks, it looks useful, do you recommend to use qt/qml for mobile development?
